docset/winserver2022-ps/smbshare/Set-SmbClientConfiguration.md

```diff
@@ -229,6 +264,27 @@ Accept pipeline input: False
 Accept wildcard characters: False
 ```
 
+### -EnableCompressibilitySampling
+Controls the sampling behavior. SMB by default always attempts to compress the entire file when a
+client or server requests it, without using compression sampling. With EnableCompressibilitySampling
+set to '$true' SMB uses of an algorithm where it attempted to compress the first 524,288,000 bytes
```

Comment:
default algorithm. Those values of 500MiB and 100MiB are what are controlled by CompressibiltySamplingSize and CompressibleThreshold above - if not populated, these are the values used (maybe that should be in their example?).

 And despite their ranges being UInt64, the _actual_ limit for the compression feature is 9,007,199,254,740,992 (9PiB). That's worth nothing on those two params help.

Reply:
I've updated this whole parameter; I've also referenced the actual compression limit in the other two parameters. Please let me know if you are happy with the updates.